website/content/docs/policies/release.mdx

```diff
@@ -0,0 +1,65 @@
+---
+layout: docs
+page_title: OpenBao Release Checklist
+sidebar_label: Release Checklist
+description: |-
+  The OpenBao checklist for releases
+---
+
+# Release checklist
+
+This checklist was created after the v2.0.0 release in order to facilitate
+a better release process going forward.
+
+## Pre-release checklist
+
+ - [ ] Clone this template into a GitHub release issue.
+ - [ ] Identify target release version (major, minor, or patch) and milestones
+       (alpha, beta, ...).
+ - [ ] Identify and communicate the release timeline.
+      - [ ] Community meeting
+      - [ ] OpenBao Core Mailing list
+      - [ ] TSC Mailing List
+      - [ ] Matrix
+      - [ ] GitHub milestones
+ - [ ] Ensure relevant features and bug fixes are present in the release.
+ - [ ] Check generated changelog via `make release-changelog` and make any
+       necessary updates to entries.
+ - [ ] Draft release notification announcement.
+ - [ ] Tag new API versions (including in auth plugins), if necessary.
+      - [ ] When bumping API major version, ensure `api/go.mod` is updated to
+            the new version.
+ - [ ] Update SDK to use the new API version, if necessary.
+      - [ ] When bumping SDK major version, ensure `sdk/go.mod` is updated to
+            the new version.
+ - [ ] Tag new SDK version, if necessary.
+ - [ ] Update first-party external dependencies to use new API & SDK
+       modules, if necessary.
+      - [ ] [`go-kms-wrapping`](https://github.com/openbao/go-kms-wrapping)
+      - [ ] [`go-secure-stdlib`](https://github.com/openbao/go-secure-stdlib)
+      - [ ] [`openbao-template`](https://github.com/openbao/openbao-template)
+ - [ ] Update API, SDK, and first-party external dependencies in main
+       `go.mod`. While API and SDK always use the built-in version, the
+       first-party external modules still use the external version and thus
+       must be updated.
+
+## Release checklist
+
+ - [ ] Tag release commit
+ - [ ] Start [release workflow](https://github.com/openbao/openbao/actions/workflows/release.yml):
+       select target tag, and set applicable pre-release/latest set
+       (pre-releases should not be marked latest on GitHub).
+ - [ ] Spot-check release artifacts:
+      - [ ] Container images work.
+      - [ ] Linux binaries work.
+      - [ ] Website download page.
+ - [ ] Open PR to update `CHANGELOG.md`.
```

Comment:
I am a fan of using the `CHANGELOG` update commit as the new version Tag, so you can directly see what changed when pulling the tag. Otherwise you need to fetch the up to date changelog from `main`.

Reply:
Ah, good idea. Then the auto-generated source code from GitHub is correct. Added :-)